File: 268-MissingNumber/268-MissingNumber.cpp
// Last updated: 9/21/2025, 10:43:37 PM
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n=nums.size();
        
        // sort(nums.begin(),nums.end());
       int total= n*(n+1)/2;

       int sum=0;

       for(int x: nums){
        sum += x;
       }
       return total-sum;
    }
};

File: 268-MissingNumber/268-MissingNumber.java
// Last updated: 11/22/2025, 11:28:26 PM
class Solution {
    public int missingNumber(int[] nums) {
        int n=nums.length;

        int total=(n*(n+1))/2;
        int sm=0;
        for(int x : nums ){
            sm +=x;
        }
        return total-sm;
    }
}